backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/workpackage/impl/AbstractMaterialReceiptLogHandler.java

```diff
@@ -128,8 +130,8 @@ public abstract class AbstractMaterialReceiptLogHandler implements IModularContr
 											.invoicingGroupId(invoicingGroupId)
 											.isBillable(isBillable)
 											.priceActual(contractSpecificPrice)
-											.amount(null)
-											.build());
+											.build()
+											.withCalculateAmount(uomConversionBL));
```

Comment:
why not simply calculate the amount here and set it?

Reply:
reduce duplicated code, because I expect it to be reused